src/main/java/com/codicesoftware/plugins/hudson/commands/ChangesetsLogCommand.java

```diff
@@ -0,0 +1,67 @@
+package com.codicesoftware.plugins.hudson.commands;
+
+import com.codicesoftware.plugins.hudson.model.ChangeSet;
+import com.codicesoftware.plugins.hudson.util.MaskedArgumentListBuilder;
+import hudson.util.Digester2;
+import org.apache.commons.digester.Digester;
+import org.xml.sax.SAXException;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ChangesetsLogCommand implements ParseableCommand<List<ChangeSet>>, Command {
+    private final String csetSpecFrom;
+    private final String csetSpecTo;
+
+    public ChangesetsLogCommand(String csetSpecFrom, String csetSpecTo) {
```

Comment:
Answering myself: I missed the return type. `ChangesetsLogCommand` returns `List<ChangeSet>` whereas `ChangesetLogCommand` returns `ChangeSet`. I'd say that names are too similar: `ChangesetLogCommand` and `ChangesetLogRangeCommand` would be probably better options to avoid confusions such as mine in the future. Do you agree? 😊

Reply:
Another option would be to still merge both classes and replace `ChangesetLogCommand` usages to check the return list size and take the first item.

Reply:
I like the name with the word `Range`. It makes it easy to understand. Renaming the classes.